extensions/Cake.SharePoint.yml

```diff
@@ -9,9 +9,9 @@ Author: JeroenVandezande
 Description: Cake AddIn to upload build artifacts to SharePoint.
 Categories:
 - sharepoint
-TargetCakeVersion: 0.38.5
+TargetCakeVersion: 1.1.0
```

Comment:
Added a comment about suggestion to target 1.0.0: https://github.com/cake-contrib/Cake.SharePoint/issues/4#issuecomment-848469018